### DISCUSSION TITLE
Slightly off topic

### DISCUSSION THREAD

**Discussion by Kristopian** [diasurgical/DevilutionX#3894] [General]
I have been absent from many forums and the like, but not because of anything big, I just have been a little busy working on a tile-set for Dwarf Fortress using DFHACK with TWBT (for a different symbol for almost every tile) and it is, well... a bit much.

**Comment by AJenbo** [comment]
Nothing special, just slowly progressing on 1.4.0, which will mostly be a settings menu and rounding off the features that had to be rushed for 1.3.0

**Comment by Kristopian** [comment]
If you see my PFP I did make it(just gonna say this because people steal other peoples art), and I also refurbished my about me.
![Screenshot 2022-01-05 1 25 33 PM](https://user-images.githubusercontent.com/91993172/148298866-35ff7580-b8cd-43bd-ab3b-2b0ba991b355.png)
Both of these accounts are mine but i forgot the password to Kristopian, so TrueKristopian it is
![Screenshot 2022-01-05 1 28 44 PM](https://user-images.githubusercontent.com/91993172/148298864-82da6633-3be1-4a0c-884e-afb82b423d92.png)
And yes I I do use pixilart.com to make pixel art(or aseprite)

**Reply by AJenbo** [reply]
This all seems really off topic, maybe join the discord and write in off-topic channel instead.

**Reply by Kristopian** [reply]
True
